### ISSUE TITLE
callback error: <class 'AttributeError'> 'str' object has no attribute 'isascii'

### ISSUE THREAD

**Issue by keneo** [randy3k/radian#421]
fresh after install:

```
$ radian
R version 3.4.4 (2018-03-15) -- "Someone to Lean On"
Platform: x86_64-pc-linux-gnu (64-bit)

r$> ls()
callback error: <class 'AttributeError'> 'str' object has no attribute 'isascii'
```

```
$ python3 --version
Python 3.7.1
$ python --version
Python 3.7.1
```

**Comment by randy3k** [comment]
Did you install radian from github? Please try again, there was a bug in the master branch.

**Comment by randy3k** [comment]
Oh, the bug actually existed in the v0.6.5, please install from github for now. I am releasing a new version to pypi.

**Comment by randy3k** [comment]
V0.6.6 is out

**Comment by keneo** [comment]
hi. it works now. thank you :)

offtopic: something interesting happened during installation (segfault):

```{bash}
(base) $ pip3 install radian                                                                                                                                                                 [7/292]
Collecting radian
  Downloading https://files.pythonhosted.org/packages/4c/d6/99be0ce0bb7fe096f5d806b46da9cea6bdbcd8cf8a53abf3ff1297b40a65/radian-0.6.6.tar.gz (54kB)
    100% |████████████████████████████████| 61kB 1.7MB/s
Collecting prompt_toolkit<3.1,>=3.0.15 (from radian)
  Using cached https://files.pythonhosted.org/packages/eb/37/791f1a6edd13c61cac85282368aa68cb0f3f164440fdf60032f2cc6ca34e/prompt_toolkit-3.0.36-py3-none-any.whl
Collecting pygments>=2.5.0 (from radian)
  Using cached https://files.pythonhosted.org/packages/0b/42/d9d95cc461f098f204cd20c85642ae40fbff81f74c300341b8d0e0df14e0/Pygments-2.14.0-py3-none-any.whl
Collecting rchitect<0.4.0,>=0.3.39 (from radian)
  Using cached https://files.pythonhosted.org/packages/9c/6c/52dca310384e5fc377e24f14f900cae1869ff17d9d7b5efd53028471a436/rchitect-0.3.40-cp36-cp36m-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.
manylinux2014_x86_64.whl
Collecting wcwidth (from prompt_toolkit<3.1,>=3.0.15->radian)
  Using cached https://files.pythonhosted.org/packages/20/f4/c0584a25144ce20bfcf1aecd041768b8c762c1eb0aa77502a3f0baa83f11/wcwidth-0.2.6-py2.py3-none-any.whl
Collecting six>=1.9.0 (from rchitect<0.4.0,>=0.3.39->radian)
  Using cached https://files.pythonhosted.org/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl
Collecting cffi>=1.10.0 (from rchitect<0.4.0,>=0.3.39->radian)
  Using cached https://files.pythonhosted.org/packages/3a/12/d6066828014b9ccb2bbb8e1d9dc28872d20669b65aeb4a86806a0757813f/cffi-1.15.1-cp36-cp36m-manylinux_2_5_x86_64.manylinux1_x86_64.whl
Collecting pycparser (from cffi>=1.10.0->rchitect<0.4.0,>=0.3.39->radian)
  Using cached https://files.pythonhosted.org/packages/62/d5/5f610ebe421e85889f2e55e33b7f9a6795bd982198517d912eb1c76e1a53/pycparser-2.21-py2.py3-none-any.whl
Building wheels for collected packages: radian
  Running setup.py bdist_wheel for radian ... done
  Stored in directory: /home/bsz/.cache/pip/wheels/b8/9a/6d/fb4d04b7e3be469b66483d4ab8f55368c53ae71c99a081e51e
Successfully built radian
Installing collected packages: wcwidth, prompt-toolkit, pygments, six, pycparser, cffi, rchitect, radian
Successfully installed cffi-1.15.1 prompt-toolkit-3.0.36 pycparser-2.21 pygments-2.14.0 radian-0.6.6 rchitect-0.3.40 six-1.16.0 wcwidth-0.2.6
Segmentation fault (core dumped)
(base) $ free -h
              total        used        free      shared  buff/cache   available
Mem:            59G         17G         26G        1.5M         14G         40G
Swap:          2.0G        2.0G          0B
(base) $ radian --version
radian version: 0.6.6
r executable: /usr/lib/R/bin/R
r version: 3.4.4
python executable: /usr/bin/python3
python version: 3.6.9
(base) $ radian
R version 3.4.4 (2018-03-15) -- "Someone to Lean On"
Platform: x86_64-pc-linux-gnu (64-bit)

r$> ls()
character(0)

r$>
```

TL;DR: it works now. thank you
